File: VB/WindowsApplication1/CustomEditor/MyViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsApplication1
	Public Class MyViewInfo
		Inherits ButtonEditViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)

		End Sub
		Protected Overrides Function CreateButtonPainter() As EditorButtonPainter
			Dim painter As New MySkinEditorButtonPainter(LookAndFeel)
			painter.OwnerEditInfo = Me
			Return painter
		End Function

	End Class
End Namespace

File: CS/WindowsApplication1/CustomEditor/MyViewInfo.cs
using System;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.Drawing;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsApplication1
{
    public class MyViewInfo : ButtonEditViewInfo
    {
        public MyViewInfo(RepositoryItem item)
            : base(item)
        {

        }
        protected override EditorButtonPainter CreateButtonPainter()
        {
            MySkinEditorButtonPainter painter = new MySkinEditorButtonPainter(LookAndFeel);
            painter.OwnerEditInfo = this;
            return painter;
        }

    }
}